src/current/_includes/releases/v23.1/v23.1.5.md

```diff
@@ -0,0 +1,93 @@
+- CockroachDB previously could encounter `zero transaction timestamp in EvalContext` when evaluating the [`COPY FROM`](../v23.1/copy-from.html) command. The bug was introduced in v23.1.0 and is now fixed. [#105295][#105295]
+- Fixed the data fetching for the Database and Table Details pages. Prior to this change, some databases/tables could be permanently stuck in a loading state, causing their corresponding page to permanently show a Spinner. This change fixes the data fetching for these pages to ensure all databases/tables are loaded correctly. [#105315][#105315]
+- Fixed a bug where [`INSERT .. ON CONFLICT .. DO UPDATE`](../v23.1/insert.html) queries incorrectly resulted in an "ambiguous column" error. The bug only presented if the target table had a computed column with an expression referencing a column with a `DEFAULT` value. [#105025][#105025]
+- On the SQL Statistics pages, a transaction query is now available as long as the statement fingerprints associated with the transaction also exist in the payload. [#105318][#105318]
+- Previously, CockroachDB would crash when evaluating [`CREATE TABLE .. AS or CREATE MATERIALIZED VIEW .. AS`](../v23.1/create-table.html) statements when the `AS` clause selected data from `crdb_internal.cluster_statement_statistics` or `crdb_internal.cluster_transaction_statistics` virtual tables. The bug has been present since at least v22.1 and is now fixed. [#105325][#105325]
+- Fixed a source of mutex contention within the storage engine that could increase tail latencies on high-CPU, high-throughput deployments. [#105361][#105361]
+- Fixed a bug where nodes could terminate with the following message: `server startup failed: cockroach server exited with error: ‹migration-job-find-already-completed›: key range id:X is unavailable: ‹failed to send RPC: no replica node information available via gossip for rX›` [#104246][#104246]
+- Previously, cross-database type references could sneak in through `CREATE TABLE...AS` statements if the source table was from another database and any of its columns was of a user-defined type. This introduced a bug where the source table could be dropped and the type could not be found for the CTAS table. This commit disallows such CTAS as a fix. [#105621][#105621]
+- CockroachDB can now automatically delete statistics for dropped tables from `system.table_statistics` table. However, out of caution, this mechanism is disabled by default on v23.1 releases. [#105479][#105479]
+- Fixed a race  condition that can occur when multiple SQLServers are created simultaneously, causing simultaneous writes to an unprotected global variable used to configure a CCL audit logging feature. [#105725][#105725]
+- Previously, referencing a user-defined type in the body of a user-defined function would result in an error at the time of creating the function. This is now fixed. [#105617][#105617]
```

Comment:
Perhaps link to ENUMs and/or user-defined functions here?

Reply:
Done.